### PR TITLE
Add my example jupyter notebook

### PR DESCRIPTION
**Pull request by elliottperryman**
I have used the linear spin wave example to write the dispersion curve calculation in jax. this has some advantages, such as being able to put some error bars on dispersion curves themselves by automatically differentiating wrt the parameters and being able to use the gradient for experimental design if spectral weights are also written.

### CODE REVIEW

**Comment by tweber-ill** [comment]
Thanks Elliott! I've moved the file to the notebooks/ folder.
